src/Assets/Scripts/GameController.cs

```diff
@@ -128,9 +130,15 @@ private void Update()
         score = value;
         scoreText.text = score.ToString();
 
-        if(score > PlayerPrefs.GetInt("HighScore"))
+        if (score > PlayerPrefs.GetInt("HighScore"))
         {
             newHighScoreText.gameObject.SetActive(true);
+
+            if(playOnce == true)
```

Comment:
This can be simplified.
```suggestion
            if(playOnce)
```
or if you use one of the variablenames i suggested.
```suggestion
            if(!hasPlayedHighScoreSound)
```
the `!` just means to flip the boolean.